openapi/components/schemas/LegacyOneTimeSale.yaml

```
@@ -0,0 +1,215 @@
+type: object
+required:
+  - orderType
+  - customerId
+  - websiteId
+  - items
+properties:
+  id:
+    type: string
+    description: ID of the order.
+    readOnly: true
+    maxLength: 50
+    example: ord_01GYJPRKHBD6ZYHH897QCJMBS4
+  orderType:
+    description: |-
+      Specifies the type of order.
+      An order may be a subscription or a one-time purchase.
+    type: string
+    x-basic: true
+    enum:
+      - one-time-order
+  customerId:
+    $ref: ./CustomerId.yaml
+  organizationId:
+    deprecated: true
+    readOnly: true
+    allOf:
+      - $ref: ./OrganizationId.yaml
+  status:
+    description: Status of the one-time order.
+    type: string
+    readOnly: true
+    enum:
+      - pending
+      - abandoned
+      - completed
+      - canceled
+  billingStatus:
+    description: |-
+      Billing status of the most recent invoice.
+      This value may help you to determine if you should change the service status of the service,
+      such as suspending the service.
+    type: string
+    readOnly: true
+    enum:
+      - draft
+      - unpaid
+      - past-due
+      - abandoned
+      - paid
+      - voided
+      - refunded
+      - disputed
+      - partially-refunded
+      - partially-paid
+  websiteId:
+    x-sortable: true
+    x-basic: true
+    allOf:
+      - $ref: ./WebsiteId.yaml
+  currency:
+    description: Currency of the order.
+    readOnly: true
+    $ref: ./CurrencyCode.yaml
+  initialInvoiceId:
+    description: ID of the initial invoice (`null` for one-time orders).
+    readOnly: true
+    type:
+      - 'string'
+      - 'null'
+    maxLength: 50
+    example: in_0YVF9605RKC62BP14NE2R7V2XT
+  recentInvoiceId:
+    description: |-
+      ID of the most recently issued invoice.
+      The invoice might not be `paid` yet.
```

suggestion: for readability
I'm not sure what the second sentence means in relation to the first. I've made a suggestion based on my understanding, which may not be correct
```suggestion
      This value is `null` if no invoice is `paid` yet.
```